auth/README.md

```diff
@@ -397,16 +397,47 @@ represented in the following diagram:
 
 ## UI customization
 
-To provide customization of the visual style of the activities that implement
-the flow, a new theme can be declared. Standard material design color
-and typography properties will take effect as expected. For example, to define
-a green theme:
+To use FirebaseUI Auth's sign-in flows, you must provide an `app_name` string and use the
+AppCompat color attributes in your app.
+
+For example, something like this goes in your `strings.xml`:
+```xml
+<resources>
+    <!-- ... -->
+    <string name="app_name">My App</string>
+</resources>
+```
+
+This goes in your `styles.xml`:
+```xml
+<style name="AppTheme" parent="FirebaseUI">
+    <!-- Required for sign-in flow styling -->
+    <item name="colorPrimary">@color/material_green_500</item>
+    <item name="colorPrimaryDark">@color/material_green_700</item>
+    <item name="colorAccent">@color/material_purple_a700</item>
+</style>
+```
+You don't have to call `setStyle()` on the sign in intent builder as FirebaseUI
+will automatically style the activities. For example:
+
+```java
+startActivityForResult(
+    AuthUI.getInstance(this).createSignInIntentBuilder()
+        .build());
+```
+
+Note: If your app is already using an AppCompat theme, you don't have to create 
+a new theme, but you can still optionally style other parts of the sign-in flow. 
+Standard material design colorand typography properties will take effect as expected. 
+For example, to define a green theme:
```

Comment:
@TheCraftKid Hmmm, the problem I see with this section is that it feels kinda like an "upgrade guide" of sorts. I'm trying to think of it from a newcomer's perspective, and for all they care, we've always auto-styled FirebaseUI so why would they have to call `setTheme()`? If you believe a newcomer would find this information useful, then feel free to keep it around.